commit 55e9b23e149b2b8a0facd0f83b0e4915b7932df5 (HEAD -> master, refs/changes/94/4494/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-08-01T14:08:09+00:00 (1 year, 2 months ago)
    
    joyent/node-vasync#40 package.json repository does not point to Joyent
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 60f6eee..c19b251 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
 	"main": "./lib/vasync.js",
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/davepacheco/node-vasync.git"
+		"url": "git://github.com/joyent/node-vasync.git"
 	},
 	"scripts": {
 		"test": "./node_modules/.bin/tap --stdout tests/ && ./node_modules/.bin/nodeunit tests/compat.js && ./node_modules/.bin/nodeunit tests/compat_tryEach.js"
